docs/tools/cli/service/connection-info.rst

```diff
@@ -0,0 +1,211 @@
+``avn service connection-info``
+==================================================
+
+Here you’ll find the full list of commands for ``avn service connection-info``.
+
+
+Retrieve connection information
+--------------------------------------------------------
+
+``avn service connection-info kafkacat``
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Retrieves the ``kcat`` :doc:`command </docs/products/kafka/howto/kcat>` necessary to connect to a certain Aiven for Apache Kafka® service.
```

Comment:
Linking the word "command" is super strange especially if you're handling links out of context. How about an additional sentence telling people "Kcat is for doing awesome with kcat, check out <our article about using this tool with Aiven>" or something? And linking back from there to here, since it removes the need for half the other article

Reply:
added, thanks